packages/fluent-theme/src/components/activity/CopilotMessageHeader.module.css

```diff
@@ -0,0 +1,42 @@
+
+:global(.webchat-fluent) .copilot-message-header {
+  align-items: center;
+  cursor: default;
+  display: flex;
+  flex-wrap: nowrap;
+  gap: var(--webchat-spacingHorizontalS);
+  margin-inline-start: var(--webchat-spacingVerticalMNudge);
+  padding-block-start: var(--webchat-spacingVerticalXS);
+  /* TODO: remove when moved to the bubble */
+  position: relative;
+  z-index: 1;
+}
+
+:global(.webchat-fluent) .copilot-message-header__avatar {
+  aspect-ratio: 1;
+  background-color: var(--background-color);
+  border-radius: var(--webchat-borderRadiusSmall);
+  width: 20px;
+}
+
+:global(.webchat-fluent) .copilot-message-header__title {
+  font-size: var(--webchat-fontSizeBase300);
+  font-weight: var(--webchat-fontWeightSemibold);
+  line-height: var(--webchat-lineHeightBase300);
+  max-width: 240px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-wrap: nowrap;
+}
+
+:global(.webchat-fluent) .copilot-message-header__ai-generated-content {
```

Comment:
Unsure if we should `nowrap` in the `__title` or here.

What if some languages display "AI Generated Content" in a very long way it drop to the second line?

Reply:
As we own translation files we could probably avoid long names. This also comes from the need to know the exact header size for paddings of the bubble...

Reply:
We don't exactly own translation files. We expect web devs to override them. :D